README.md

```diff
@@ -1,3 +1,4 @@
 Testing a local repo creation
 
-# Adding section to represent local development of instructions branch
\ No newline at end of file
+
```

Comment:
Added extra line during the marge just to see how it would get marked down here in the file diff